### PR TITLE
Fix: Gitter Icon color in the footer

### PR DESCRIPTION
**Pull request by Vani-j01**
Fixed the gitter icon color in the footer of index.html

### CODE REVIEW

File: images/svg/gitter-brands.svg [hunk 1]
```diff
@@ -1 +1,7 @@
-<svg aria-hidden="true" focusable="false" data-prefix="fab" data-icon="gitter" class="svg-inline--fa fa-gitter fa-w-12" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 384 512"><path fill="currentColor" d="M66.4 322.5H16V0h50.4v322.5zM166.9 76.1h-50.4V512h50.4V76.1zm100.6 0h-50.4V512h50.4V76.1zM368 76h-50.4v247H368V76z"></path></svg>
\ No newline at end of file
+<svg aria-hidden="true" focusable="false" data-prefix="fab" data-icon="gitter" class="svg-inline--fa fa-gitter fa-w-12 " role="img" 
+xmlns="http://www.w3.org/2000/svg" viewBox="0 0 384 512">
+<!--Added color to the icon using fill-->
+<path fill="#007bff" 
+ d="M66.4 322.5H16V0h50.4v322.5zM166.9 76.1h-50.4V512h50.4V76.1zm100.6 0h-50.4V512h50.4V76.1zM368 76h-50.4v247H368V76z">
+ </path>
+ </svg>
```
Comment (cclauss) [inline images/svg/gitter-brands.svg]: Please add one blank line at the end of the file so GitHub gets rid of that ⛔ sign.

Comment (Panquesito7) [reply images/svg/gitter-brands.svg]:
```suggestion
 </svg>
 
```

**Review by Panquesito7 — APPROVED** [review]
👍 apart from the above.

File: images/svg/gitter-brands.svg (hunk 1 above)
Comment (Panquesito7) [reply images/svg/gitter-brands.svg]: The error is still present.

Comment (Vani-j01) [reply images/svg/gitter-brands.svg]: I did add the new line. But the symbol seems to stay at the end of the file no matter how many extra lines  I add. It was there in the original file I forked too.

Comment (Vani-j01) [reply images/svg/gitter-brands.svg]: @Panquesito7  I have removed the symbol in the last commit. Can you please review it for me. This is my first PR so I might be a bit sloppy apologies for that.

**Comment by Panquesito7** [comment]
Ping?

**Comment by mrmaxguns** [comment]
I'll close this PR since TheAlgorithms has a new website.

**Comment by Panquesito7** [comment]
@mrmaxguns the website is still present, but it is deprecated. You can still go to the old website by going to <https://thealgorithms.github.io/index-old.html>.

**Comment by Panquesito7** [comment]
https://github.com/TheAlgorithms/TheAlgorithms.github.io/commit/05b2571834f713e9372db4ee1aa564b33bff28db

**Comment by mrmaxguns** [comment]
I understand, but I see no reason to continue improving it, and servicing it since it's deprecated.

**Comment by Panquesito7** [comment]
No more improvements and changes will be made now. **It shouldn't be archived (see the message at [Gitter](https://gitter.im/TheAlgorithms/community?at=603a8d54e8267a46f2dec26b)).**

**Comment by mrmaxguns** [comment]
Yes, I will keep that in mind. But this pull request specifically addresses the Gitter icon which is only found in `index-old.html`. Thanks for letting me know. I wasn't planning on archiving the repository :smiley:

**Pull request closed without merging**